src/ape/types/vm.py

```diff
@@ -14,12 +14,12 @@
 A type that represents contract code, which can be represented in string, bytes, or HexBytes.
 """
 
-SnapshotID = Union[str, int, bytes]
+SnapshotID = Any
 """
 An ID representing a point in time on a blockchain, as used in the
 :meth:`~ape.managers.chain.ChainManager.snapshot` and
-:meth:`~ape.managers.chain.ChainManager.snapshot` methods. Can be a ``str``, ``int``, or ``bytes``.
-Providers will expect and handle snapshot IDs differently. There shouldn't be a need to change
-providers when using this feature, so there should not be confusion over this type in practical use
-cases.
+:meth:`~ape.managers.chain.ChainManager.snapshot` methods. Can be a ``str``, ``int``, ``bytes``,
```

Comment:
for whatever reasons, we have been sticking to `1 sentence per line` in our markdown, that is why you see such long lines. We can change this but probably should be all changed at once.